### ISSUE TITLE
Missing directory functionality, folder workspace  

### ISSUE THREAD

**Issue by sxzxs** [aardappel/treesheets#774]
Missing directory functionality, folder workspace  

This function can open multiple directories, making it easy to switch notes.

**Comment by aardappel** [comment]
I don't think we plan to add a "folder workspace", if you mean something akin to what VSCode has.

The idea is to keep multiple notes inside a single TreeSheet document.
